From 7b1c78779d6c01e376d540613509f1a43a4780ca Mon Sep 17 00:00:00 2001
From: Yaniv Levinsky <ylevinsky@habana.ai>
Date: Thu, 20 Jun 2024 12:55:23 +0300
Subject: [PATCH] [PNP-174] api: update tdp input validation

The API input validation will limit TDP inputs above bit 11 (2048W).

Scale validation managed to reach 2100W peaks in their torture tests, so
we should ease this limit to allow better tests.

Change the limit to bit 12 (4096W).

Change-Id: I725a024ea4d7aa0c4865a67b43435ffdb2abef37
Signed-off-by: Yaniv Levinsky <ylevinsky@habana.ai>
---

diff --git a/include/lib/api.h b/include/lib/api.h
index d5cb50e..bb15863 100644
--- a/include/lib/api.h
+++ b/include/lib/api.h
@@ -15,7 +15,7 @@
 #define PID_API_VERSION_MINOR		2
 #define PID_API_VERSION_PATCH		0
 
-#define PID_API_MAX_TDP			BIT(11)
+#define PID_API_MAX_TDP			BIT(12)
 #define PID_API_MAX_TRIP_POINT		BIT(11)
 #define PID_API_MAX_EDP_FACTOR		100
 #define PID_API_MAX_EDP_WINDOW		0xf
